census/estimator/preprocessing/preprocess.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Defines the Beam pipeline to preprocess the data."""
+
+import csv
+
+import apache_beam as beam
+import tensorflow as tf
+
+
+class CsvFileSource(beam.io.filebasedsource.FileBasedSource):
+
+  def read_records(self, file_name, range_tracker):
```

Comment:
`range_tracker` appears unused. You can rename it to `unused_range_tracker` if it's needed in the signature, or write the following in the function body:
```
_ = range_tracker
```